app/controllers/mod/stories_controller.rb

```diff
@@ -16,7 +16,7 @@ def edit
   def update
     @story.is_deleted = false
     @story.editor = @user
-    @story.attributes = story_params
+    update_story_attributes
```

Comment:
I would rather repeat the standard Rails idiom you removed than wrap up that single line of code in a function.